api/v2/serializers/details/resource_request.py

```diff
@@ -14,7 +14,7 @@ class ResourceRequestSerializer(serializers.HyperlinkedModelSerializer):
     )
     created_by = UserSummarySerializer(read_only=True)
     identity = IdentityRelatedField(source='membership.identity')
-    status = StatusTypeRelatedField(allow_null=True, required=False)
```

Comment:
Because `StatusTypeRelatedField` creates a serializer in it's custom `to_representation`, for 400 resources requests 400 serializers were constructed. This doubled the time contributed by this one field.